schemas/io-package.json

```diff
@@ -0,0 +1,457 @@
+                        }
+                     },
+                    "additionalProperties": false
+                },
+                "desc": {
+                    "description": "Description, object for multilingual description",
+                    "type": "object",
+                    "required": ["en"],
+                    "patternProperties": {
+                        "en|de|ru|pt|nl|fr|it|es|pl|zh-cn": {
+                            "type": "string"
+                        }
+                    },
+                    "additionalProperties": false
+                },
+                "messagebox": {
+                    "description": "true if messagebox supported. If yes, the object system.adapter.<instance>.messagebox will be created to send messages to adapter",
+                    "type": "boolean"
+                },
+                "readme": {
+                    "description": "[DEPRECATED]: Use docs",
+                    "type": "string"
+                },
+                "docs": {
+                    "description": "The structure like {\"en\": \"docs/en/README.md\", \"de\": [\"docs/de/README.md\", \"docs/de/README1.md\"]} that describes the documentation if not in README.md",
+                    "type": "object",
+                    "required": ["en"],
+                    "patternProperties": {
+                        "en|de|ru|pt|nl|fr|it|es|pl|zh-cn": {
+                            "type": "string"
+                        }
+                    },
+                    "additionalProperties": false
+                },
+                "authors": {
+                    "description": "Array of authors as string",
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    }
+                },
+                "license": {
+                    "description": "License of the software",
+                    "type": "string"
+                },
+                "type": {
+                    "description": "Type of the adapter",
+                    "type": "string",
+                    "enum": ["alarm", "climate-control", "communication", "date-and-time", "energy", "metering", "garden", "general",
+                        "geoposition", "hardware", "health", "household", "infrastructure", "iot-systems", "lighting", "logic", "messaging",
+                        "misc-data", "multimedia", "network", "protocols", "storage", "utility", "visualization", "visualization-icons",
+                        "visualization-widgets", "weather"]
+                },
+                "unsafePerm": {
+                    "description": "if the package must be installed with \"npm --unsafe-perm\" parameter",
+                    "type": "boolean"
+                },
+                "plugins": {
+                    "description": "ioBroker plugins to be used",
+                    "type": "object"
```

Comment:
Can we be more specific here?

Reply:
Hm every plugin is very specific, like sentry properties will not be used anywhere else

```
        "plugins": {
            "sentry": {
                "dsn": "https://ab26a840557947c088448fead588e0c5@sentry.iobroker.net/9",
                "pathWhitelist": [
                    "@iobroker",
                    "iobroker.js-controller"
                ]
            }
        }
        ```